packages/components/data-table/src/header-cell.styles.tsx

```diff
@@ -116,16 +118,19 @@ const BaseHeaderCell = styled.th<TBaseHeaderCell>`
    * - not using "border-bottom" since it stands out in front of the resize indicator and counts towards the row height
    * - not using "box-shadow" since it's already used for the column divider
```

Comment:
```suggestion
```
nit: since the column divider is removed in this PR, I would remove this comment as well

Reply:
updated [5016b9a](https://github.com/commercetools/ui-kit/pull/2452/commits/5016b9ad798f2a01fd5813a7ada06df689ffb0c3)